adapters/IronSource/IronSourceAdapter.xcodeproj/project.pbxproj

```diff
@@ -79,26 +85,32 @@
-		4584560E22AEE59A0061764B /* GADMAdapterIronSourceRewardedAd.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = GADMAdapterIronSourceRewardedAd.h; sourceTree = "<group>"; };
-		4584560F22AEE59A0061764B /* GADMediationAdapterIronSource.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GADMediationAdapterIronSource.m; sourceTree = "<group>"; };
-		4584561022AEE59A0061764B /* ISMediationManager.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = ISMediationManager.m; sourceTree = "<group>"; };
-		4584561122AEE59A0061764B /* GADMAdapterIronSourceRewarded.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GADMAdapterIronSourceRewarded.m; sourceTree = "<group>"; };
-		4584561222AEE59B0061764B /* GADMAdapterIronSource.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GADMAdapterIronSource.m; sourceTree = "<group>"; };
-		4584561322AEE59B0061764B /* GADMAdapterIronSourceUtils.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = GADMAdapterIronSourceUtils.h; sourceTree = "<group>"; };
-		4584561422AEE59B0061764B /* GADMAdapterIronSourceUtils.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GADMAdapterIronSourceUtils.m; sourceTree = "<group>"; };
-		4584561522AEE59B0061764B /* GADMAdapterIronSourceInterstitialDelegate.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = GADMAdapterIronSourceInterstitialDelegate.h; sourceTree = "<group>"; };
 		7D46CFDF1DB6B7CC0050F612 /* Adapter.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Adapter.xcconfig; sourceTree = "<group>"; };
 		7D6A68A31DB7ED230001E48B /* Debug.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Debug.xcconfig; sourceTree = "<group>"; };
 		7D6A68A41DB7ED310001E48B /* Release.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Release.xcconfig; sourceTree = "<group>"; };
 		7D9552B11DC006E5005DCE8D /* Script_Framework.sh */ = {isa = PBXFileReference; lastKnownFileType = text.script.sh; path = Script_Framework.sh; sourceTree = "<group>"; };
 		7DFABB1D1DA8682C00322E02 /* libAdapter.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = libAdapter.a; sourceTree = BUILT_PRODUCTS_DIR; };
+		EFD8A00D2A695CBF00EB90DA /* GADMAdapterIronSourceUtils.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; name = GADMAdapterIronSourceUtils.h; path = ../../../../../Downloads/IronSourceAdapter/GADMAdapterIronSourceUtils.h; sourceTree = "<group>"; };
```

Comment:
I think these should be reverted as it makes the project files undetectable since it is expecting the files to be under `/Downloads`

Reply:
Resolved